src/index.ts

```diff
@@ -82,6 +82,7 @@ function bjorklund(subsequences: any[][]) {
  * @returns The array of evenly mixed booleans
  */
 export function euclid(numberOfTrue: number, numberOfFalse: number): boolean[] {
+  // return bresenham(numberOfTrue, numberOfFalse, true, false);
```

Comment:
You forgot to actually use your new method.

Reply:
Normally you shouldn't add commented pieces of code, but I guess this documents how to do the benchmarking so it's fine for now.